python/sparkdl/transformers/named_image.py

```diff
@@ -140,7 +141,16 @@ def __call__(self, value):
 _scaleHintConverter = _LazyScaleHintConverter()
 
 
-class DeepImageFeaturizer(JavaTransformer, HasInputCol, HasOutputCol):
+class _DeepImageFeaturizerReader(JavaMLReader):
+    def __init__(self, clazz):
+        super(_DeepImageFeaturizerReader, self).__init__(clazz)
+
+    @classmethod
+    def _java_loader_class(cls, clazz):
+        return "com.databricks.sparkdl.DeepImageFeaturizer"
```

Comment:
with the default implementation, is the problem that we don't get `com.databricks.sparkdl`, just `sparkdl` ?

Reply:
We get sparkdl.transformers.named_image.DeepImageFeaturizer, the paths are almost completely different.